WooCommerce/Classes/ViewRelated/Editor/FormatBar/TextList+AztecFormatting.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import Aztec
+
+// MARK: - TextList.Style
+//
+extension TextList.Style {
+
+    var formattingIdentifier: FormattingIdentifier {
+        switch self {
+        case .ordered:   return FormattingIdentifier.orderedlist
+        case .unordered: return FormattingIdentifier.unorderedlist
+        }
+    }
+
+    var description: String {
+        switch self {
+        case .ordered: return "Ordered List"
+        case .unordered: return "Unordered List"
+        }
+    }
+
+    var accessibilityLabel: String {
```

Comment:
Also here, It might make sense to also add accessibilityTraits and accessibilityHint, to improve accessibility like in the rest of the app. I’m sorry for the duplicate comments!